### ISSUE TITLE
traces export: failed to send to http://coroot:8080/v1/traces: 404 Not Found

### ISSUE THREAD

**Issue by kr521** [coroot/coroot-node-agent#103]
<img width="649" alt="image" src="https://github.com/coroot/coroot-node-agent/assets/3262751/3f3abae3-0ff5-4a94-9024-bd8ef01e4723">

<img width="931" alt="image" src="https://github.com/coroot/coroot-node-agent/assets/3262751/e730643e-6b8e-4533-a189-749533b1123a">

agent version: 1.20.3
kernel version: 5.14.0-284.25.1.el9_2.sinarelease.6.x86_64
containerd://1.7.12
k8s version: v1.28.7

install with helm

**Comment by kr521** [comment]
profiling.go:152] failed to upload 404: project not found

**Comment by kr521** [comment]
coroot-cluster-agent:1.0.3

**Comment by kr521** [comment]
<img width="886" alt="image" src="https://github.com/coroot/coroot-node-agent/assets/3262751/19fb07d6-393b-40b9-a7ad-076f4b9b3743">

The newly created service has no issues. However, after adding a new project, the node-agent starts reporting a 404 error. Once the newly added project is deleted, the error no longer occurs.

**Comment by u-kyou** [comment]
> The newly created service has no issues. However, after adding a new project, the node-agent starts reporting a 404 error. Once the newly added project is deleted, the error no longer occurs.

Yes, I encountered the same problem when configuring multiple projects.

**Comment by skob** [comment]
any news here?
same behavior when renaming default project :-(
